xbmc/GUIInfoManager.cpp

```diff
@@ -4556,6 +4556,16 @@ std::string CGUIInfoManager::GetItemLabel(const CFileItem *item, int info, std::
         return dateTime.GetAsLocalizedDate();
       break;
     }
+    if (item->HasEPGInfoTag())
```

Comment:
should we also provide this for channel and recording items too? see LISTITEM_GENRE